jacobian_follower/src/jacobian_follower/jacobian_follower.cpp

```diff
@@ -274,11 +274,14 @@ std::optional<moveit_msgs::RobotState> JacobianFollower::computeCollisionFreePoi
     ROS_DEBUG_STREAM_NAMED(LOGGER_NAME + ".ik",
                            "goal:  [" << name << "] positions " << p.x << "," << p.y << "," << p.z);
   }
+  opts.goals.emplace_back(std::make_unique<bio_ik::MinimalDisplacementGoal>());
 
   robot_state::RobotState robot_state_ik(model_);
+  robot_state::RobotState seed_robot_state_ik(model_);
```

Comment:
Do we have a style guide for initialization? Any reason why you are using `(..)` initialization instead of `{...}` initialization? One advantage of `{...}` initialization is avoiding the [most vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse)